### ISSUE TITLE
CNV wdl might be out of date wrt cromwell

### ISSUE THREAD

**Issue by davidbenjamin** [broadinstitute/gatk-protected#934]
I was having problems with the `getBamFileName` task in the cnv workflow running cromwell 25.  The task would finish without proceeding to the next task.  This happened running on a Broad VM with and without SGE.

The task looks like
```java
command {
    echo something
}

output {
    String result = read_string(stdout())
}
```

the problem goes away when I switch to
```java
command {
    echo something > "tmp"
}

output {
    String result = read_string("tmp")
}
```

I'm having the same problem of tasks finishing but the workflow hanging with no cromwell error elsewhere in the workflow, but I can't diagnose them.

**Comment by davidbenjamin** [comment]
@asmirnov239 This is what I was talking about.

**Comment by meganshand** [comment]
Could this be a regression in v25?

**Comment by davidbenjamin** [comment]
Possible -- this wdl worked a month or two ago, on cromwell 22 or 23, maybe 24.

**Comment by asmirnov239** [comment]
This won't be a problem after closing #820. But we should probably figure out why it's happening and maybe file an issue in Cromwell repo

**Comment by samuelklee** [comment]
Can confirm this was working fine in v24.

**Comment by samuelklee** [comment]
Closed in #983, which got rid of the getBamFilename task.
